doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -538,12 +594,12 @@ gls{component}
 \labelwidthstring 00.00.0000
 \begin_inset CommandInset label
 LatexCommand label
-name "/OF120/"
+name "/OF180/"
 
 \end_inset
 
  If requested by the user, Beagle shuts down the computer it's running on
- after it finished.
+ after it finished a measurement.
```

Comment:
This sentence doesn't makes sense. It should be something like "If reqeusted by the user, Beagle shuts down the computer when/after a measurment has been finished."